apps/oc_chef_wm/itest/oc_chef_wm_keys_SUITE.erl

```diff
@@ -277,6 +489,7 @@ make_user(Config, Name, AuthzId) ->
                                      {<<"display_name">>, <<"someone">>}]}),
     chef_db:create(User, context(), ?USER_AUTHZ_ID).
 
+%% TODO: should this be updated to use the POST endpoint?
```

Comment:
In the cases we're using it we just want the data there - when we want to test behavior of the POST endpoint we do it directly. 

That said,  there's something to be said for minimizing the amount of  SQL that we have to embed in our tests.